### ISSUE TITLE
create parent tooth terms

### ISSUE THREAD

**Issue by megbalk** [obophenotype/uberon#2759]
**Preferred term label:**
add parent terms like:

incisor tooth 3
incisor tooth 4
incisor tooth 5
molar 5

**Synonyms**

**Definition (free text, please give PubMed ID)**
The incisor tooth of the upper or lower jaw that is phylogenetically number 3 or 4 or 5.

The molar tooth of the upper or lower jaw that is phylogenetically number 5.

**Parent term (use https://www.ebi.ac.uk/ols/ontologies/uberon)**
incisor tooth
molar tooth

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-2699-3066

**Comment by megbalk** [comment]
@wdduncan was curious about 1) using the term phylogenetically and wanting to express it differently, and 2) using numbers as some users may not be familiar with this way.

**Comment by megbalk** [comment]
@bvarner-ebi @cmungall any suggestions?

**Comment by megbalk** [comment]
We can put 'has_exact_synonym': 'first incisor tooth' as an alternative for using numbers.

**Comment by bvarner-ebi** [comment]
Hi, @megbalk,

Consider identifying what numbering system or standard you are using to assign the numbers (e.g., [Universal Numbering System](https://web.archive.org/web/20061102074427/http://www.ada.org/public/topics/tooth_number.asp), Triadan system, etc.).

If you are not using a standardised numbering system, consider stating how you are assigning the numbers (e.g., the tooth in the second position counted starting from the upper, most posterior tooth).

**Comment by wdduncan** [comment]
@bvarner-ebi Do you know of tooth numbering systems that are cross species? I've only seen the number systems (e.g, Universal, FDI) used for humans.

**Comment by wdduncan** [comment]
> Modified Triadan appears to be cross-species.

Interesting ... I wasn't aware that system :)

**Comment by megbalk** [comment]
@bvarner-ebi @wdduncan I don't think either system is being used here. I'll take the suggestion to clarify how these are counted. Should I create an issue to clarify for all teeth?

**Comment by wdduncan** [comment]
@megbalk Perhaps you can use labels like `first homologous incisor tooth`? Does this makes it clear that the numbering is baed on homology and not tooth position?

**Comment by megbalk** [comment]
Would that be a synonym? I am hesitant to rename _all_ the teeth...

**Comment by wdduncan** [comment]
Only for those tooth classes in which the axis of classification is homology.

**Comment by megbalk** [comment]
I think in Uberon that is all of them? Like molar tooth 1, molar tooth 2, etc. which already exist. It would require us to redo all those terms. I think I assumed "phylogenetically" meant something along the same lines of homologous?

**Comment by wdduncan** [comment]
The issue I think is that the label "molar tooth 1" may be too ambiguous (given the different tooth nomenclatures). My suggestion was meant to disambiguate the label. 

For example, because I've used the ADA tooth numbering system for too long, when I see "molar tooth 1", I think of the upper-right wisdom tooth. 

Also, do some of the tooth labels with numbers in them refer to teeth positionally instead of phylogenetically? For example, `pharyngobranchial 2 tooth` is defined as `Tooth that is attached to the pharyngobranchial 2 tooth plate`, and ` pharyngobranchial 2 tooth plate` is defined as `Pharyngeal tooth plate that lies ventral to pharyngobranchial 2 bone`. In this definition, is `lies ventral to` meant to refer to the spatial location (or position)?

**Comment by bvarner-ebi** [comment]
@megbalk, re:

>It's based on an idea of homology for a tooth, where the first tooth isn't arbitrary, it's the same tooth type that emerges in that first position.

The issue seems to be how "first" is defined in a term like "first position". Is this counted spatially (if so, what is the coordinate framework) or developmentally/temporally? My recommendation is to clarify what counting system is used somewhere in the class (definition, comment).

**Comment by megbalk** [comment]
@bvarner-ebi got it, can do. Should I add that to all the parent tooth terms (i.e., ones already in existence)?

**Comment by bvarner-ebi** [comment]
> @bvarner-ebi got it, can do. Should I add that to all the parent tooth terms (i.e., ones already in existence)?

Adding to all parent terms with a numbered position (new and existing) seems reasonable. I think doing that would obviate potential confusion.

**Comment by megbalk** [comment]
how is this for the revised definition: 

The molar tooth of the upper or lower jaw that is phylogenetically and developmentally in position 1, counting from the rostral-most to the caudal-most.

**Comment by wdduncan** [comment]
Does "developmentally in position 1" mean that it is the first tooth of that type to erupt in the dentition and arch?

**Comment by cmungall** [comment]
@megbalk is correct, the number system is phylogenetic. This is the same for other serial homologs in Uberon. We get less confused when it's the digits as humans have the ancestral digits.

Apologies that this is not made clearer, and also it's not entirely consistent due to different editors following different practice:

```yaml
id: UBERON:0018302
name: upper molar 1
def: "First molar (counting anteroposteriorly) in the upper jaw." [PHENOSCAPE:Alex]

id: UBERON:0018376
name: molar tooth 1
def: "The molar tooth of the upper or lower jaw that is phylogenetically number 1." [UBERON:cjm]
```

We should stick with the phylogenetic system since Uberon is multispecies, but we can change the specific wording if it's done consistently. E.g. "molar tooth 1 (phylogenetic)". We should also place each such term into a DP.

I realize that these names are not ideals for users who want a human anatomy, but the solution here is to tag the preferred synonym, then when we make the human view we can substitute these for the primary label. E.g.

```yaml
id: UBERON:0018377
name: molar tooth 3
def: "The molar tooth of the upper or lower jaw that is phylogenetically number 3." [UBERON:cjm]
...
synonym: "wisdom tooth" RELATED HUMAN_PREFERRED [NCBITaxon:9606]
```

**Comment by wdduncan** [comment]
@cmungall Yes. I understand your point about UBERON being multi-species and that the labels shouldn't be necessarily geared towards human anatomy. My question concerned clarification about the meaning of "phylogenetically number 1". I'm not familiar with phylogenetic numbering. Does it mean the first molar tooth to appear in the phylogenic tree?

**Comment by bvarner-ebi** [comment]
> We should stick with the phylogenetic system since Uberon is multispecies,

@cmungall, even if it Uberon covers multiple species, can you provide context as to why numbering a tooth phylogenetically opposed to a anatomical position is preferred? Anterior, rostral, caudal, etc are species-agnostic terms as far as I know and would not require a user to know about evolutionary development in order to use the ontology. 

To @wdduncan's point above, does a phylogenetic numbering system exist? I could not find one after a cursory online search, so will it be clear to users what the difference is from "phylogenetically number 1" and "phylogenetically number 2", as an example?

**Comment by megbalk** [comment]
@bvarner-ebi the reason to name them phylogenetically and not positionally is because some species can lack a specific tooth or teeth. For example, camels lack P1 and P2. If you call the first premolar one encounters P1 on a camel, that is confusing as it is actually the same as P3 in other mammals.

**Comment by megbalk** [comment]
I don't think we ever came to a conclusion, but I vote for keeping these parent tooth terms consistent with tooth terms that already exist.

**Comment by wdduncan** [comment]
No ... we didn't come to a conclusion. 
The positional information is important, but runs into issues for cross species representation. However, not having positional information, would limit the utility of using UBERON for studies about teeth.

**Comment by megbalk** [comment]
So this doesn't close, can we have the parent tooth terms follow the current pattern?

**Comment by wdduncan** [comment]
Lot's of conversation in this thread.

What is the current pattern you are referring to? Do you mean phylogenic numbering? If so, then I think it is important to include 'phylogenic' in the label.

**Comment by megbalk** [comment]
yes, keeping "phylogenetic".

**Comment by meghalithic** [comment]
not at the moment, but think it should be done eventually.

**Comment by meghalithic** [comment]
@rays22 I need guidance from someone. Should we use "phylogenetically" like has always been used, or change to something more understandable, like "homologous"?
